python/py-tmuxp/Portfile

```diff
@@ -2,15 +2,18 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
+PortGroup           select 1.0
 
 name                py-tmuxp
-version             1.7.0
+version             1.7.2
+revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
In the first commit, I update `py-tmux`, and the next commit I add support for the select port. It's why do I use the revision. Maybe does it not affect since this is one pull-request?

Reply:
That is not relevant. My comment still stands. The `revision` should be 0.

Reply:
The `revision` only gets incremented in specific cases. This is not one of them.

Reply:
Please read the [wiki reference](https://guide.macports.org/#reference.keywords) for it for the specifics.

Reply:
Thanks for the explanation.